.github/workflows/ghcr.yml

```
@@ -3,6 +3,7 @@ name: Publish Docker Image
 on:
   push:
     branches: [ '**' ]
+    tags: [ '**' ]
```

This could probably just be:
```suggestion
on:
  push:
```